architecture/core_concepts/routes.adoc

```diff
@@ -621,6 +622,11 @@ the destination. This field is required, but only for re-encryption.
 [[router-sharding]]
 == Router Sharding
 
+An OpenShift administrator can deploy multiple routers in an OpenShift cluster. Since the
+routers always listen on ports 80 (http) and 443 (https), the routers are placed on
```

Comment:
That's not true.  You can change the ports with the ROUTER_SERVICE_HTTP_PORT and ROUTER_SERVICE_HTTPS_PORT environment variables.  They just default to 80 and 443 respectively.